python/podio/test_ReaderSio.py

```diff
@@ -3,7 +3,10 @@
 
 import unittest
 
-from podio.sio_io import Reader, LegacyReader
+try:
+  from podio.sio_io import Reader, LegacyReader
+except ImportError:
+  print('Unable to load SIO for this unit test, aborting the test...')
```

Comment:
There is already a facility in place to skip sio based tests, when podio is not built with sio support. So here, rather than catching the `ImportError` I would just `import` the respective reader in the respective test case.